Core/Transports/MongoDBTransport/MongoDB.cs

```diff
@@ -21,6 +21,7 @@ public enum Field
 
 // Question: the benefit of noSQL is the use of unstructured collections of variable documents.
 // Explore storing partially serialized Speckle objects with dynamically generated fields instead of just a content string?
+[Obsolete("This Transport is no longer maintained or routinely tested, use with caution")]
```

Comment:
Should discuss this as a team before merge, and check how many users are using this (if any).
We haven't tested this in over a year. I think we should either re-vitalise it, or deprecate it. 

I may compile a list of some other parts of core we could consider deprecating (serializerV1) but we'll need to make sure all the team has had the chance to voice their thoughts.